clients/mobile/__tests__/authentication/useLogin.test.ts

```diff
@@ -1,19 +1,65 @@
-import { mockedNavigate, setRouteParams } from '../__mocks__/reactNavigation.mock'
 import { renderHook, act } from '@testing-library/react-native'
 import { useLogin } from '../../src/authentication/login/hooks/useLogin'
 import { loginUser, googleLogin, facebookLogin } from '../../src/authentication/services/authService'
+import { SCREENS } from '../../src/setup/constant/screens'
+
+const mockDispatch = jest.fn()
+const mockSetIsAuthenticated = jest.fn()
+let mockUserProfile: { bloodGroup?: string } | null = null
+
+jest.mock('@react-navigation/native', () => ({
+  ...jest.requireActual('@react-navigation/native'),
+  useNavigation: () => ({
+    dispatch: mockDispatch,
+    navigate: jest.fn()
+  }),
+  CommonActions: {
+    reset: (config: any) => config
+  }
+}))
```

Comment:
We do have a mock setup for `@react-navigation/native`, so why do you again mock this?

Reply:
As you might already know, we are temporarily skipping unit tests until the first release. I have added placeholders for them just to ensure the CI passes. According to Khirul Vai and Shuaib Vai, we will schedule dedicated time after the first release to write all the missing tests and optimize the existing ones. I hope this clarifies the situation.

Reply:
vai, let's not worry about unit test for now